src/main/java/net/minecraftforge/event/entity/EntityFinalizeSpawnEvent.java

```diff
@@ -0,0 +1,55 @@
+package net.minecraftforge.event.entity;
+
+import javax.annotation.Nullable;
+
+import net.minecraft.nbt.CompoundTag;
+import net.minecraft.world.DifficultyInstance;
+import net.minecraft.world.entity.Mob;
+import net.minecraft.world.entity.MobSpawnType;
+import net.minecraft.world.entity.SpawnGroupData;
+
+/**
+ * This event is called when an entity finalizes its spawn, usually to give it something (eg, skeletons with bows)<br>
+ * This event should be used if a modder wants to add something to an entity the first time it's spawned.<br>
+ * If a modder wants to add something everytime an entity is spawned they should use {@link EntityJoinWorldEvent}
```

Comment:
Rather then `everytime an entity is spawned`, it may be more understandable if it were reworded to `everytime an entity is added to the world`, as "spawned" is more likely to be understood in the context of "being spawned by something, like a spawner, spawn egg, etc."